apps/ledger-live-mobile/src/components/DeviceAction/rendering.tsx

```diff
@@ -579,6 +585,91 @@ export function renderError({
   );
 }
 
+export function RequiredFirmwareUpdate({
+  t,
+  device,
+  navigation,
+}: RawProps & {
+  // Sorry for the any DX team 🙇
+  navigation: any;
```

Comment:
as this is a function component you could remove it from props and just get it with `useNavigation()` -> no more any needed